### ISSUE TITLE
The order of the custom workflow parameters list changed.

### ISSUE THREAD

**Issue by jakechai** [Acly/krita-ai-diffusion#1401]
The order of the parameters list was based on the node ID of ComfyUI, which is reasonable for me.
But after the latest update, the order became unrecognizable.
Or, is it necessary to add custom priority for Krita parameter nodes?

![微信截图_20241114160359](https://github.com/user-attachments/assets/79ce203c-efc8-4d8f-af35-8e7d84b045de)

**Comment by Acly** [comment]
Yes it's sorted now, I think alphabetically if you don't specify custom order?

Using the creation order from the workflow wasn't very practical: when you wanted to insert a node in the middle or change the order, you'd have to recreate all of them in the UI.

Using `1. `, `2. ` etc. in front of the name should work properly now.
